code/__DEFINES/role_preferences.dm

```diff
@@ -42,7 +42,9 @@
 #define ROLE_FAMILIES "Familes Antagonists"
 #define ROLE_SYNDICATE_CYBERSUN "Cybersun Space Syndicate" //Ghost role syndi from Forgottenship ruin
 #define ROLE_SYNDICATE_CYBERSUN_CAPTAIN "Cybersun Space Syndicate Captain" //Forgottenship captain syndie
```

Comment:
and while you're here if you could add a slash to these and put them above the defines so they count as docs, that'd be fantastic